### ISSUE TITLE
relabel image only works with python lists and not numpy lists

### ISSUE THREAD

**Issue by Cryaaa** [haesleinhuepf/napari-skimage-regionprops#33]
When using the relabel image function it only works with python lists because the function tries to add the first element of the predictionlist the pythonic way with a `+`:

```python
def relabel_cle(image, measurements):
    import pyclesperanto_prototype as cle
    return cle.pull(cle.replace_intensities(image, numpy.asarray([0] + measurements)))

def relabel_numpy(image, measurements):
    return numpy.take(numpy.array([0] + measurements), image)
```

Instead we could reform the measurement list like this:
```python
return numpy.take(np.insert(np.array(measurement), 0, 0))
```

This would make the function compatible with python as well as numpy lists.

**Comment by haesleinhuepf** [comment]
Hey Ryan @Cryaaa ,

yes, this makes a lot of sense. In the pyclesperanto-case, we could make use of `cle.prefix_in_x` to put the `[0]` in front of the measurements (available since pyclesperanto_prototype 0.18.2).
 Do you want to send a PR? Otherwise, I'll do it by the end of the week.

Thanks for raising the issue btw!

**Comment by Cryaaa** [comment]
I can make a quick PR no problem. Maybe its best to stick to numpy for this library though as it doesn't have a clesperanto dependency.
